package.json

```
@@ -72,14 +72,14 @@
     "prepare-for-release": "terra prepare-for-release",
     "pretest": "npm run lint",
     "postinstall": "link-parent-bin && npm run compile && npm run bootstrap:hoist",
-    "start": "tt-serve",
-    "start-heroku": "tt-serve-static --port $PORT --site './build'",
-    "start-prod": "tt-serve --env.disableHotReloading -p",
-    "start-static": "tt-serve-static --site ./build",
+    "start": "webpack-dev-server",
+    "start:prod": "webpack-dev-server --env.disableHotReloading -p",
+    "start-static": "npm run compile:prod && terra express-server --site ./build",
+    "start-heroku": "terra express-server --port $PORT --site './build' ",
     "test": "npm run lint && npm run jest && npm run wdio",
-    "wdio-default": "wdio",
-    "wdio-lowlight": "THEME=clinical-lowlight-theme wdio",
-    "wdio-fusion": "THEME=orion-fusion-theme wdio",
+    "wdio-default": "terra wdio",
+    "wdio-lowlight": "terra wdio --themes clinical-lowlight-theme",
+    "wdio-fusion": "terra wdio --themes orion-fusion-theme wdio",
     "wdio": "npm run wdio-default && npm run wdio-lowlight && npm run wdio-fusion"
```

one thing to note - once we've fixed the cli/env issue, we can update this script to pass in an array of themes
```suggestion
    "wdio": "terra wdio  --themes terra-default-theme, clinical-lowlight-theme, orion-fusion-theme"
```